Admin/PageAdmin.php

```diff
@@ -432,19 +432,19 @@ protected function configureSideMenu(MenuItemInterface $menu, $action, AdminInte
         $id = $admin->getRequest()->get('id');
 
         $menu->addChild('sidemenu.link_edit_page',
-            array('uri' => $admin->generateUrl('edit', array('id' => $id)))
+            $admin->generateMenuUrl('edit', array('id' => $id))
         );
 
         $menu->addChild('sidemenu.link_compose_page',
-            array('uri' => $admin->generateUrl('compose', array('id' => $id)))
+            $admin->generateMenuUrl('compose', array('id' => $id))
         );
 
         $menu->addChild('sidemenu.link_list_blocks',
-            array('uri' => $admin->generateUrl('sonata.page.admin.page|sonata.page.admin.block.list', array('id' => $id)))
+            $admin->generateMenuUrl('sonata.page.admin.block.list', array('id' => $id))
         );
 
         $menu->addChild('sidemenu.link_list_snapshots',
-            array('uri' => $admin->generateUrl('sonata.page.admin.page|sonata.page.admin.snapshot.list', array('id' => $id)))
+            $admin->generateMenuUrl('sonata.page.admin.snapshot.list', array('id' => $id))
```

Comment:
can you please indent the `//` in line 457 while you are on this file right now?